apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/create-a-site.md

```diff
@@ -0,0 +1,167 @@
+---
+title: 'Tutorial: How to Create a Site and Connect your App to it'
```

Comment:
On my screen it puts the "it" into the next row which does not look good. Maybe title can just be: "Tutorial: How to Create a Site and Connect your App"

Reply:
Great call, and it reads better too. I never liked that "to it" part. Thank you! The commit change is done, deploying it now.